src/detectors/transpilers/html/parser.ts

```diff
@@ -60,7 +60,10 @@ export async function extractJSScriptTags(contentStream: ReadStream) {
 				// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type#attribute_is_not_set_default_an_empty_string_or_a_javascript_mime_type
 				return attr.name.value !== "type" ||
 					(attr.name.value === "type" &&
-					(attr.value.value === "" || attr.value.value === "text/javascript"));
+					["",
+						"text/javascript",
+						"application/javascript", /* legacy */
```

Comment:
Please also add a test for `application/javascript`

Reply:
Done